### ISSUE TITLE
Make samples browsable in dev mode

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor-sdk#159]
Currently if I try to load: http://myhost/ckeditor-sdk/samples/sharedspace.html

I get 404 errors because path to CKEditor is `vendor/ckeditor/` and there's no CSS too.

It'd be good if path that we have in the source HTML files would be correct for the dev version. They will need to be updated during building SDK.

**Comment by Reinmar** [comment]
Hm... But as I think about this now, we would need to use the `../ckeditor-dev/` path for CKEditor. We have this convention that every repo is in the same directory so this would work. We actually already require this in ckeditor-docs so I guess we should be ok here.

In other words samples should have this:

```
	<script src="../../ckeditor-dev/ckeditor.js"></script>
```

instead of:

```
	<script src="../vendor/ckeditor/ckeditor.js"></script>
```

**Comment by Tade0** [comment]
I found that there's more to this than adjusting a few paths. The build script, among other things, inserts content into the sidebar.

I'm investigating whether it's possible to do this more-or-less painlessly.

**Comment by Reinmar** [comment]
Fixed by #160. Good work!
